Api Web/Models/Department.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Api_Web.Models
+{
+    public class Department
+    {
+        [Key]
+        public int DepartmentEmployeeId { get; set; }
```

Comment:
everything seems to be in good shape however we could change the variable to Id, or EmployeeId, something more understandable for other devs.